substrate/client/service/src/lib.rs

```diff
@@ -27,10 +27,7 @@ pub mod config;
 pub mod error;
 
 mod builder;
-#[cfg(feature = "test-helpers")]
 pub mod client;
```

Comment:
nit: could be grouped with other pub mods.

Reply:
Done :+1: